### PR TITLE
[4.0] Namespace privacy modules

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Namespaces privacy modules.

### Testing Instructions

Apply patch. Delete `libraries/autoload_psr4.php` file.
Publish `mod_latestactions`, `mod_privacy_dashboard` and `mod_privacy_status` modules in backend.
Inspect the modules.

### Expected result

Works like before.

### Documentation Changes Required

IDK.

### CODE REVIEW

File: administrator/modules/mod_latestactions/mod_latestactions.php [hunk 1]
```diff
@@ -10,21 +10,19 @@
 defined('_JEXEC') or die;
```
Comment (Quy) [inline administrator/modules/mod_latestactions/mod_latestactions.php:10]:
```suggestion
\defined('_JEXEC') or die;
```

Comment (SharkyKZ) [reply administrator/modules/mod_latestactions/mod_latestactions.php:10]: No need because file is not namespaced.

**Comment by vaibhavsTekdi** [comment]
I have tested this item :white_check_mark: successfully on f5da9e54c25030df111e94bb738d04e5aa302637<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26240">issues.joomla.org/tracker/joomla-cms/26240</a>.</sub>

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on f5da9e54c25030df111e94bb738d04e5aa302637<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26240">issues.joomla.org/tracker/joomla-cms/26240</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26240">issues.joomla.org/tracker/joomla-cms/26240</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**